src/components/form.tsx

```diff
@@ -69,15 +85,15 @@ export class Form extends React.Component<FormProps, FormState> {
             <div className="form__item">
                 {React.cloneElement(child, errors && {
                     validationError: errors,
-                    submitted
+                    submitted: child.props.submitted || submitted
```

Comment:
Will this crash if submitted is not a property on the child?

Reply:
No, `child.props.submitted` would be undefined, and so it would use `submitted`, which by default is false